lib/template/lib/endpoints/schema.rb

```diff
@@ -0,0 +1,19 @@
+module Endpoints
+  class Schema < Base
+    get "/schema.json" do
+      content_type("application/schema+json")
+      response.headers["Cache-Control"] = "public, max-age=3600"
+      filename = schema_filename
+      unless File.exists?(filename)
+        raise Pliny::Errors::NotFound.new("Schema")
```

Comment:
What do you think about something slightly more descriptive for this one? Maybe something like "This application does not have a schema file."